exp-player/addon/components/exp-overview/component.js

```diff
@@ -29,6 +29,12 @@ var generateValidators = function(questions) {
   return validators;
 };
 
+var getQuestion8Options = function() {
+    var scale = range(1, 10);
+    scale.push('survey.sections.1.questions.8.options.preferNoAnswer');
```

Comment:
Hmm, might want to tweak that before merge then, so that no bad sample data makes it into production.